zebrad/tests/start_no_args.rs

```diff
@@ -0,0 +1,34 @@
+use color_eyre::eyre::Result;
+use std::time::Duration;
+use zebra_test::prelude::*;
+
+pub fn get_child_single_arg(arg: &str) -> Result<(zebra_test::command::TestChild, impl Drop)> {
```

Comment:
TIOLI: At this point I don't think there's much value to having this broken out into its own function, I'd inline this code back into the `start_no_args` fn personally.

Reply:
Yea, this was actually just for testing the CI, i had the plan of making this functions callable from here(or just inline it) however as separating by now didn't make any difference this split can be reverted.

Reply:
aah yea, makes sense. I'm gonna pull this down locally and play around with it a bit.

Reply:
Okay so I got an okay initial solution I think. At least good enough to unblock this. I combined those three tests into a single test like so:

```rust
#[test]
fn serialized_tests() -> Result<()> {
    start_no_args()?;
    start_args()?;
    seed_no_args()?;

    Ok(())
}
```

This has the disadvantage that we will only see one item in the test output from `cargo` but I think that's okay for now.

Reply:
gonna push a commit that adds this

Reply:
You are the best!